server-nodejs/extensions/jwt_delegate.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * jwt_delegate.js
+ */
+ 
+module.exports = function (app) {
+    let db  = app.locals.db;
+    let damas = require ('damas-core-client');
+    let crypto = require('crypto');
+    damas.server = app.locals.conf.jwt_delegate.server;
+    
+    app.use('/api/signIn', function (req, res, next) {
+        if ('undefined' != req.body.username) {
+            damas.signIn(req.body.username, req.body.password, function (user) {
+                if (undefined === user || null === user) {
+                    console.warn("Connexionn with the server : ", damas.server, " impossible.");
```

Comment:
«Connexionn» is something like a french word

Reply:
Sorry for this syntax error.